src/core/core.layouts.js

```diff
@@ -155,32 +332,13 @@ module.exports = {
 	 * @param {number} height - the height to fit into
 	 */
 	update: function(chart, width, height) {
-		if (!chart) {
```

Comment:
what's the history of this check? Is it safe to remove?

Reply:
I could only track it that far: https://github.com/chartjs/Chart.js/pull/1750
Anyway, this seems to be publicly available through `Chart.layouts`, so external code could call it without chart. I'll restore this check.